coalib/bearlib/abstractions/Linter.py

```diff
@@ -207,9 +207,9 @@ def get_executable():
             Returns the executable of this class.
 
             :return:
-                The executable name.
+                The full path to the executable file.
             """
-            return options['executable']
+            return shutil.which(options['executable'])
```

Comment:
maybe just assign `shutil.which(executable)` to `options['executable']`, below at `def linter(...)`?